installscripts/jazz-terraform-unix-noinstances/scripts/DeleteStackPlatformServices.py

```diff
@@ -41,20 +41,21 @@ def deleteCloudFormationService(service_name):
 platformServices = ['jazz_cognito-authorizer', 'jazz_logs', 'jazz_usermanagement', 'jazz_services-handler', 'jazz_events', 'jazz_services', 'jazz_logout', 'jazz_login', 'jazz_cloud-logs-streamer', 'jazz_is-service-available', 'jazz_delete-serverless-service', 'jazz_create-serverless-service', 'jazz_email', 'jazz_events-handler', 'jazz_environments', 'jazz_scm-webhook' , 'jazz_environment-event-handler', 'jazz_deployments', 'jazz_deployments-event-handler']
 
 for pservice in platformServices:
-    deleteCloudFormationService(stackName + pservice + '-prod')
+    deleteCloudFormationService(stackName + pservice)
```

Comment:
We need `-prod`, bad merge probably?

Reply:
Dangit, you're right, fixed this and rechecked every conflict from that merge.